### PR TITLE
Big Bench Audio Release

### PR DESCRIPTION
**Pull request by mhillsmith**
Big Bench Audio blog post ready to go hopefully!

Congratulations! You've made it this far! Once merged, the article will appear at https://huggingface.co/blog. Official articles 
require additional reviews. Alternatively, you can write a community article following the process [here](https://huggingface.co/blog-explorers).

## Preparing the Article

You're not quite done yet, though. Please make sure to follow this process (as documented [here](https://github.com/huggingface/blog/tree/main?tab=readme-ov-file#how-to-write-an-article-)):

- [x ] Add an entry to [_blog.yml](https://github.com/huggingface/blog/blob/main/_blog.yml).
- [ x] Add a thumbnail. There are no requirements here, but there is a [template](https://github.com/huggingface/blog?tab=readme-ov-file#how-to-get-a-nice-responsive-thumbnail) if it's helpful.
- [x ] Check you use a short title and blog path.
- [x ] Upload any additional assets (such as images) to the Documentation Images [repo](https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog). This is to reduce bloat in the GitHub base repo when cloning and pulling. Try to have small images to avoid a slow or expensive user experience.
- [ x] Add metadata (such as authors) to your `md` file. You can also specify `guest` or `org` for the authors.
- [ x] Ensure the publication date is correct.
- [ x] Preview the content. A quick way is to paste the markdown content in https://huggingface.co/new-blog. **Do not click publish**, this is just a way to do an early check.

Here is an example of a complete PR: https://github.com/huggingface/blog/pull/2382

## Getting a Review

Please make sure to get a review from someone on your team or a co-author.
Once this is done and once all the steps above are completed, you should be able to merge.
There is no need for additional reviews if you and your co-authors are happy and meet all of the above.

Feel free to add @pcuenca as a reviewer if you want a final check. Keep in mind he'll be biased toward light reviews
(e.g., check for proper metadata) rather than content reviews unless explicitly asked.

### CODE REVIEW

File: big-bench-audio-release.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+---
+title: "Evaluating Audio Reasoning with Big Bench Audio" 
+thumbnail: /blog/assets/101_decision-transformers-train/thumbnail.gif
+authors:
+- user: mhillsmith
```
Comment (Vaibhavs10) [inline big-bench-audio-release.md:5]:
```suggestion
- user: mhillsmith
  guest: true
  org: ArtificialAnalysis
```

File: big-bench-audio-release.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+- user: georgewritescode
```
Comment (Vaibhavs10) [inline big-bench-audio-release.md:6]:
```suggestion
- user: georgewritescode
  guest: true
  org: ArtificialAnalysis
```

File: big-bench-audio-release.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+
+To support analysis of this, Artificial Analysis is releasing **[Big Bench Audio](https://huggingface.co/datasets/ArtificialAnalysis/big_bench_audio)**, a new evaluation dataset for assessing the reasoning capabilities of audio language models. This dataset adapts questions from **[Big Bench Hard](https://arxiv.org/pdf/2210.09261)** - chosen for its rigorous testing of advanced reasoning - into the audio domain.
+
+This post introduces the Big Bench Audio dataset alongside initial benchmark results for GPT-4o and Gemini 1.5 series models. Our analysis examines these models across multiple modalities: native Speech to Speech, Speech to Text, Text to Speech and Text to Text. We present a summary of results below, and on the new Speech to Speech page on the **Artificial Analysis** website: [**https://artificialanalysis.ai/models/speech-to-speech**](https://artificialanalysis.ai/models/speech-to-speech).
+
```
Comment (Vaibhavs10) [inline big-bench-audio-release.md:16]: It might be nice to add a TL;DR here about the model performance.

File: big-bench-audio-release.md [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+## The Big Bench Audio Dataset
+
+Big Bench Audio comprises **1,000 audio questions** selected from four categories of Big Bench Hard, each chosen for their suitability for audio evaluation:
```
Comment (Vaibhavs10) [inline big-bench-audio-release.md:19]:
```suggestion
[Big Bench Audio](https://huggingface.co/datasets/ArtificialAnalysis/big_bench_audio) comprises **1,000 audio questions** selected from four categories of Big Bench Hard, each chosen for their suitability for audio evaluation:
```

relinking it at the start for ease of use.

**Review by Vaibhavs10 — APPROVED** [review]
looking good, left some suggestions to make it pack a punch a bit more 🔥

File: big-bench-audio-release.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+
+- **Output:**
+
+  <audio controls="controls">
+    <source type="audio/wav" src="https://huggingface.co/datasets/ArtificialAnalysis/hf-assets/resolve/main/question_607-answer.wav?download=true"></source>
+  </audio>
+
+- **Is correct:** `True`
+
+### Text to Text
```
Comment (pcuenca) [inline big-bench-audio-release.md:212]: Not sure it's necessary to provide examples for the 4 different setups. Your call, though!

File: big-bench-audio-release.md [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+
+All results presented represent averages across three independent evaluation runs on each dataset. 
+
+### The Audio Reasoning Gap
+
+<center>
+
+
+</center>
+
```
Comment (pcuenca) [inline big-bench-audio-release.md:249]:
```suggestion
```

File: big-bench-audio-release.md [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+Our analysis reveals a substantial performance gap between text reasoning and audio reasoning. GPT-4o (Aug '24) achieves 92% accuracy in the Text to Text version of the dataset, while its Speech to Speech counterpart (GPT-4o Realtime Preview Oct '24) reaches a score of 66%. The Text to Speech configuration achieves intermediate performance at 74%, indicating that both speech input and speech output are contributing to the performance gap.
```
Comment (pcuenca) [inline big-bench-audio-release.md:250]: super interesting. Could part of this be explained by transcription mistakes or inaccuracies?

File: big-bench-audio-release.md [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+
+
+### Speech to Speech Pipelines Currently Outperform Native Audio for Reasoning
+
+<center>
+
+</center>
+
```
Comment (pcuenca) [inline big-bench-audio-release.md:258]:
```suggestion
```

File: big-bench-audio-release.md (hunk 3 above)
Comment (pcuenca) [reply big-bench-audio-release.md:16]: Very much agree, anticipating that the native support lags behind text or traditional pipelines is a very interesting takeaway.

**Review by pcuenca — APPROVED** [review]

File: big-bench-audio-release.md [hunk 9]
```diff
@@ -0,0 +1,268 @@
...
+- An input audio file is provided and the model generates an output audio file containing the answer.
+
+**Speech to Text**
+- An input audio file is provided and the model generates a text answer.
+
+**Text to Speech**
+- A text version of the question is provided and the model generates an output audio file containing the answer.
+
+**Text to Text**
+- A text version of the question is provided and the model generates a text answer.
```
[the remark this answers is not in the document]
Comment (mhillsmith) [reply big-bench-audio-release.md:65]: Yep good idea, I'll make that change!

File: big-bench-audio-release.md (hunk 5 above)
Comment (mhillsmith) [reply big-bench-audio-release.md:202]: Leaving in to give readers a a clear sense of the different ways these models can be used - a bit of education alongside introducing the dataset is the thinking!

File: big-bench-audio-release.md (hunk 7 above)
Comment (mhillsmith) [reply big-bench-audio-release.md:240]: Somewhat - the speech output gap could be about transcription but there are a couple of facts indicating that not to be the case:

- The whisper/4o/OpenAI TTS pipeline approach didn't see the same drop - near equal performance to text to text
- On manual review, we didn't identify any transcription problems

File: big-bench-audio-release.md [hunk 10]
```diff
@@ -0,0 +1,268 @@
...
+## Examples
+
+Below are some expamples responses.
+
+### Speech to Speech
+
+- **Model:** GPT-4o, Realtime Preview
+- **Input:**
+  
+  <audio controls="controls">
```
[the remark this answers is not in the document]
Comment (mhillsmith) [reply big-bench-audio-release.md:130]: Unless that's a strong view, my view is basically that there are enough text examples to read as-is and making readers listen to a couple questions to properly understand what we're asking the models to do is quite helpful!

**Comment by mhillsmith** [comment]
@pcuenca @Vaibhavs10 I think I've worked through all the suggestions - appreciate the input! Good to go on my end now if you guys area.

**Pull request merged**